vault/identity_store_aliases.go

```diff
@@ -203,7 +200,7 @@ func (i *IdentityStore) handleAliasUpdateCommon() framework.OperationFunc {
 			// If we can look up by the given canonical ID, see if this is a
 			// transfer; otherwise if we found no previous entity but we find one
 			// here, use it.
-			canonicalEntity, err := i.MemDBEntityByID(canonicalID, true)
+			canonicalEntity, err := i.MemDBEntityByID(canonicalID, false)
```

Comment:
I'm curious why this was flipped to false.

Reply:
It was something I was trying while investigating an issue and had forgot to flip back. You pointed out a potential bug. Thanks! Fixed it.